packages/xo-web/src/common/xo/migrate-vms-modal/index.js

```diff
@@ -262,6 +262,11 @@ export default class MigrateVmsModalBody extends BaseComponent {
   _toggleSmartVifMapping = () =>
     this.setState({ smartVifMapping: !this.state.smartVifMapping })
 
+  compareContainers = (pool1, pool2) => {
+    const poolIds = map(this.props.vms, '$pool')
+    return poolIds.includes(pool1.id) ? -1 : poolIds.includes(pool2.id) ? 1 : 0
+  }
```

Comment:
This does not work correctly because the compare function has an implicit dependency on `this.props.vms`, which means that the pools will not be reorder if the VMs change but not the pools.

Reply:
In Home page we select multiple VMs and we migrate'em as bulk. I don't see how VMs would change !

Reply:
So what have been in this previously validated [PR](https://github.com/vatesfr/xen-orchestra/commit/4ce702dfdf23fbf11304ca11bec2d0cb94573e28#diff-fac9535c85de22599d12e3068c33f47cR217) is wrong too !